### DISCUSSION TITLE
Development build

### DISCUSSION THREAD

**Discussion by isivisi** [isivisi/Chess.com#1] [General]
This is a work in progress version of a more feature rich version of the original chess.com app code.

features:
- You now get notifications when the board state changes on the current page. You will get all board notifications in the event view as well.
- You now have a settings page were you can specify if you want notifications, discord presence, open on startup, and other things in the future.

Issues atm: 
- chessboard notification settings for live, online, or events don't actually do anything right now.
- notifications on out of focus does not work, only minimized works
- board notifications are a frame behind the actual move, event board notifications are off centered
- the current size of the screen effects notification placement a bit

<hr /><em>This discussion was created from the release <a href='https://github.com/isivisi/Chess.com/releases/tag/v1.1.0'>Development build</a>.</em>

**Comment by JairusSW** [comment]
Great! Thank you!
